ci/playbooks/deploy_watcher_service.yaml

```
@@ -8,6 +8,9 @@
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
   tasks:
+    - name: Import tasks for creating COO subscription
+      ansible.builtin.include_tasks: "create-coo-subscription.yaml"
+
```

I can see the similar playbook is added here: https://github.com/openstack-k8s-operators/telemetry-operator/blob/main/ci/create-coo-subscription.yaml. We can also reuse the playbook. It is just a suggestion.
```suggestion
    - name: Import tasks for creating COO subscription
      ansible.builtin.include_tasks: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/telemetry-operator/ci/create-coo-subscription.yaml"
```
Note, we also need add `openstack-k8s-operators/telemetry-operator` under required-projects in watcher-operator-base job.

```
required-projects:
   - openstack-k8s-operators/telemetry-operator
```

---

We are going with make target approach so resolving it.